README.md

```diff
@@ -120,6 +122,19 @@ lib/linguist.rb
 …
 ```
 
+#### Single file
+
+Alternatively you can find stats for a single file using the `github-linguist` executable.
+
+You can try running `github-linguist` on files in this repository itself:
+
+```console
+$ bundle exec bin/github-linguist grammars.yml
```

Comment:
Whooops. I've just realised we detail global gem installation instructions and then suggest using bundler to exec which won't work.

Mind changing this to:

```suggestion
$ github-linguist grammars.yml
```

... and whilst you're at it, would you mind doing the same in the "Git Repository" section above? 🙇‍♂️